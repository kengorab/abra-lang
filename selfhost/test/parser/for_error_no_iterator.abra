for s in { } val a = 1
