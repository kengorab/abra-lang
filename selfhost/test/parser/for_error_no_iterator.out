Error at %FILE_NAME%:1:14
Unexpected token 'val':
  |  for s in { } val a = 1
                  ^